openassessment/xblock/student_training_mixin.py

```diff
@@ -49,8 +49,8 @@ def render_student_training(self, data, suffix=''):   # pylint:disable=W0613
 
         try:
             path, context = self.training_path_and_context()
-        except: # pylint:disable=W0702
-            msg = u"Could not render student training step for submission {}".format(self.submission_uuid)
+        except:  # pylint:disable=W0702
+            msg = u"Could not render Learner Training step for submission {}".format(self.submission_uuid)
```

Comment:
These are errors messages recorded in the system logs. I'll add a period to the end of the sentence.